web/src/composebox_typeahead.ts

```diff
@@ -89,56 +156,56 @@ export function update_emoji_data(initial_emojis) {
     }
 }
 
-export function topics_seen_for(stream_id) {
+export function topics_seen_for(stream_id?: number): string[] {
     if (!stream_id) {
         return [];
     }
 
     // Fetch topic history from the server, in case we will need it soon.
-    stream_topic_history_util.get_server_history(stream_id, () => {});
+    stream_topic_history_util.get_server_history(stream_id, () => {
+        /* Don't do anything with the results for now.  */
```

Comment:
A better comment would be "We'll use these extended results in the next keypress, but avoid trying to live-update what's already shown, which can be annoying."